gotham-server/src/storage/aws/dynamodb.rs

```diff
@@ -0,0 +1,231 @@
+use std::thread;
+use std::time::Duration;
+use std::default::Default;
+use serde_json;
+use serde;
+use rusoto_core::{ProvideAwsCredentials, DispatchSignedRequest};
+use rusoto_dynamodb::*;
+use super::error::*;
+use super::*;
+use std::string::*;
+use std::collections::HashMap;
+use std::fmt::Debug;
+
+#[derive(Serialize, Deserialize)]
+struct DBItem<T> {
+    id: String,
+    v: T,
+}
+
+pub fn insert<T>(dynamodb_client: &DynamoDbClient, id: &str, table_name: &str, v: T) where T: serde::ser::Serialize {
+    let wrapper_item = DBItem {id: id.to_string(), v};
+    let item = serde_dynamodb::to_hashmap(&wrapper_item).unwrap();
+    let put_item_input = PutItemInput {
+        item: item,
+        table_name: table_name.to_string(),
+        ..Default::default()
+    };
+    // Put item
+    let put_item_result = dynamodb_client.put_item(put_item_input).sync();
```

Comment:
This does not generate a compiler warning? you are not processing the result here

Reply:
Done. Now handling errors from the DB.